### ISSUE TITLE
Error importing the lib

### ISSUE THREAD

**Issue by Tomas-caula** [react-native-share/react-native-share#1296]
Hi I am trying to use the library but when I import it I get this error

I'm importing this with `import Share from 'react-native-share';`

```
"null is not an object (evaluating _reactNative.NativeModules.RNShare.FACEBOOK)"
ativeModules.RNShare.FACEBOOK)"
```

What's going on here?

**Comment by mikehardy** [comment]
Standard react-native "native module not integrated correctly" error.
If the javascript bundle is telling you the native module is null, that means it is not built in to the app for whatever reason.
Either you are using Expo in some way that doesn't do native modules or you did not pod install (you don't even specify a platform! https://stackoverflow.com/help/how-to-ask) or you did not rebuild or auto-linking is broken.

Bottom line: Not a bug in this module. Bug in your project + native module integration
